docs/code_navigation/explanations/index.mdx

```diff
@@ -1,17 +1,3 @@
 # Explanations
 
-- [Introduction to code navigation](/code_navigation/explanations/introduction_to_code_navigation)
-- [Precise code navigation](/code_navigation/explanations/precise_code_navigation)
-  - [Code graph data uploads](/code_navigation/explanations/uploads)
-- [Search-based code navigation](/code_navigation/explanations/search_based_code_navigation)
-- [Code navigation features](/code_navigation/explanations/features)
-  - [Popover](/code_navigation/explanations/features#popover)
-  - [Go to definition](/code_navigation/explanations/features#go-to-definition)
-  - [Find references](/code_navigation/explanations/features#find-references)
-  - [Dependency navigation](/code_navigation/explanations/features#dependency-navigation) (Beta)
-  - [Find implementations](/code_navigation/explanations/features#find-implementations)
-  - [Symbol search](/code_navigation/explanations/features#symbol-search)
-- [Rockskip: faster search-based code navigation](/code_navigation/explanations/rockskip) (Beta)
-- [Writing an indexer](/code_navigation/explanations/writing_an_indexer)
-- [Auto-indexing](/code_navigation/explanations/auto_indexing)
-- [Auto-indexing inference](/code_navigation/explanations/auto_indexing_inference)
+- [Code graph data uploads](/code_navigation/explanations/uploads)
```

Comment:
Could you elaborate on why all of these links are being deleted? Are we abandoning the Divio documentation architecture?